src/mines.js

```diff
@@ -0,0 +1,397 @@
+let win;
+let timerVar;
+let amountOfFlags;
+let amountOfMines;
+let seconds = 0;
+let minutes = 0;
+let end;
+let rows;
+let columns;
+let checkpoint = true;
+let bombed = false;
+
+function remove(arr, value) {
+  for (let i = 0; i < arr.length; i += 1) {
+    if (arr[i] === value) {
+      arr.splice(i, 1);
+    }
+  }
+}
+
+function getRandomInt(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function checkIfHasMines(i1, i2) {
+  if (
+    document.querySelector(`[data-order = "${i1}_${i2}"]`) !== null &&
+    parseInt(
+      document
+        .querySelector(`[data-order = "${i1}_${i2}"]`)
+        .getAttribute("data-mined"),
+      2
+    ) === 1
+  ) {
+    return true;
+  }
```

Comment:
You perform `querySelector` twice here for the same item. This should be optimised.